main.tf

```diff
@@ -35,6 +35,14 @@ resource "google_container_cluster" "cluster" {
 
   cluster_ipv4_cidr = var.cluster_ipv4_cidr
 
+  dynamic "authenticator_groups_config" {
+    for_each = var.rbac_security_identity_group != null ? [1] : []
```

Comment:
just FYI, I've just kept the variable in the variables.tf in the previous version that why its not in this PR